packages/expo-cli/src/credentials/context.ts

```diff
@@ -41,6 +43,9 @@ export class Context {
   get api(): ApiV2 {
     return this._apiClient as ApiV2;
   }
+  get android(): AndroidApi {
+    return this._androidApiClient as AndroidApi;
```

Comment:
Hmmm, I don't like this `as AndroidApi`. Maybe it's worth adding a conditional to check if `this._androidApiClient` is undefined? And then throw some error if it's undefined.

Reply:
Yes, I know we don't have such conditionals in other getters in this class.

Reply:
those getters are only unassigned if the context was not initialized

Reply:
I added api client when called outside the project context